buildenv/jenkins/JenkinsfileBase

```diff
@@ -169,7 +169,11 @@ def post(test_target) {
 }
 
 def testBuild() {
-	timeout(time: 6, unit: 'HOURS') {
+	def time_limit = 8
+	if(params.TIME_LIMIT) {
```

Comment:
Will this cause "MissingPropertyException"?

Reply:
no, this use of `params.XXX` will not cause that kind of exception, verified on private Jenkins

Reply:
Ok, thanks for clarify.